test/integration/targets/openssh_keypair/tasks/main.yml

```diff
@@ -27,4 +27,45 @@
     path: '{{ output_dir }}/privatekey5'
   register: publickey_gen
 
+- name: Generate privatekey6
+  openssh_keypair:
+    path: '{{ output_dir }}/privatekey6'
+    type: rsa
+
+- name: Regenerate privatekey6 via force
+  openssh_keypair:
+    path: '{{ output_dir }}/privatekey6'
+    type: rsa
+    force: yes
+  register: output_regenerated_via_force
+
+- name: Create broken key
+  copy:
+    dest: '{{ item }}'
+    content: ''
+    mode: '0700'
+  loop:
+    - '{{ output_dir }}/privatekeybroken'
+    - '{{ output_dir }}/privatekeybroken.pub'
+
+- name: Regenerate broken key
+  openssh_keypair:
+    path: '{{ output_dir }}/privatekeybroken'
+    type: rsa
+    force: yes
```

Comment:
```suggestion
```
I would remove the `force` here. The module is supposed to regenerate invalid keys automatically, without the need of `force`. The `force` option should only be needed to regenerate keys which fit all parameters.